db/schema.rb

```diff
@@ -15,6 +15,21 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "delayed_jobs", force: :cascade do |t|
+    t.integer  "priority",   default: 0, null: false
+    t.integer  "attempts",   default: 0, null: false
+    t.text     "handler",                null: false
+    t.text     "last_error"
+    t.datetime "run_at"
+    t.datetime "locked_at"
+    t.datetime "failed_at"
+    t.string   "locked_by"
+    t.string   "queue"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.index ["priority", "run_at"], name: "delayed_jobs_priority", using: :btree
+  end
```

Comment:
Instead of adding a table that we're not going to use (at least not yet), can we not just remove the delayed jobs functionality?

Looks like it should just be removing the `delayed_job_active_record` gem and [line 25](https://github.com/zapdigital/taskopolis/blob/master/config/application.rb#L25) in `config/application.rb`.